preorder.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>Sandstorm</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=1100px">
+    <meta property="og:title" content="Sandstorm.io">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="https://sandstorm.io/logo-bw.png">
+    <meta property="og:url" content="https://sandstorm.io">
+    <meta property="og:description"
+        content="Take control of your web by running your own personal cloud server with Sandstorm.">
+    <link rel="stylesheet" type="text/css" href="preorder.css">
+  </head>
+
+  <body>
+    <nav>
+      <a href="/">Home</a>
+      <a href="https://demo.sandstorm.io">Demo</a>
+      <a href="https://blog.sandstorm.io">News</a>
+      <a href="https://github.com/sandstorm-io/sandstorm">GitHub</a>
+      <a href="https://groups.google.com/group/sandstorm-dev">Dev Group</a>
+    </nav>
+
+    <header>
+    </header>
+
+    <div class="modal" id="modal-background"></div>
+    <div class="modal" id="modal-content">
+      <div id="modal-header">
+        <div id="modal-header-box">
+          <h3>Sandstorm.io</h3>
+          <h4>EARLY BIRD PRE-ORDER</h4>
+        </div>
+      </div>
+      <div id="modal-body">
+        <p id="title">
+          <b>Sandstorm will always be open source</b> and you're free to<br />
+          host it on your own linux box for yourself and your friends.
+        </p>
+        <p id="textbody">
+          But not everyone wants to carry a pager or has the time and experience to maintain their own<br />
+          instance, so we're offering the convenience of managed hosting, which you can pre-order.
```

Comment:
Actually let's change this line to:

But not everyone wants to use their own machine or has the time and experience to maintain a Linux server, so we're offering the convenience of managed hosting, which you can pre-order.